src/DSi_DSP.h

```diff
@@ -64,7 +64,7 @@ class DSi_DSP
     void IrqRep1();
     void IrqRep2();
     void IrqSem();
-    u16 DSPRead16(u32 addr);
+    u16 DSPRead16(u32 addr) const;
```

Comment:
I'm not familiar with the DSP address space, but MMIO where reading triggers an action is a thing.